helm/tmpnotes/templates/redis.yaml

```diff
@@ -0,0 +1,77 @@
+{{ if .Values.redis.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: redis
+spec:
+  progressDeadlineSeconds: 600
+  replicas: 1
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      app: redis
+  strategy:
+    rollingUpdate:
+      maxSurge: 25%
+      maxUnavailable: 25%
+    type: RollingUpdate
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        app: redis
+    spec:
+      automountServiceAccountToken: false
+      containers:
+        - command:
+            - redis-server
+            - --bind
+            - 0.0.0.0
+            - --dir
+            - /bitnami/redis/data
+          image: bitnami/redis:6.2
```

Comment:
+1 for using the bitnami redis image and making this part of the chart enabled with a value! :tada: 

Could we mount a volume so the data persists between redis restarts? 

Could we set this to the latest image/redis 7?

Reply:
Do you think it would be worth helm randomly generating a secret to use for auth?

Reply:
interesting thoughts:
1. Volumes shouldn't be a requirement, optionally mounting for persistence sounds like a good idea.
2. Random secret / inject your own secret would be good I think.
Thinking about it more, potentially just using the bitnami/redis helm chart? https://bitnami.com/stack/redis/helm

Reply:
@kwsorensen I thought about this a bit, and went with the approach to just use a single instance redis pod with persistence. We should just say in the docs that this does not provide a HA redis deployment, but you are free to use your own redis instance (maybe that helm chart you linked) and override the `TMPNOTES_REDIS_URL` env var. Do you think that makes sense? I just mostly don't want to be on the hook for providing a robust redis installation with this project because there are so many ways to do that and it could end up being quite a bit of maintenance for us.

Reply:
I'd suggest we at least give an option to not be persistent. IMO it's tmpnotes... persistence shouldn't be the default - but definitely not a requirement.